pipeline/pipeline.go

```diff
@@ -115,6 +115,7 @@ type Settings struct {
 
 // New creates new pipeline. Consider using `SetupHTTPHandlers` next.
 func New(name string, settings *Settings, registry *prometheus.Registry) *Pipeline {
+	fmt.Println("create new pipeline")
```

Comment:
Forgotten debug statement. If not, change it to `logger.Info("creating new pipeline")`

Reply:
sure, removing. thanks!